class.git.php

```diff
@@ -343,6 +343,34 @@ public function pull($path, $remote, $branch) {
             $result = $this->executeCommand($command);
             return $this->parseShellResult($result, "Repository pulled!", "Failed to pull repo!");
         }
+                
+        public function fetch($path, $remote) {
+            if (!is_dir($path)) return $this->returnMessage("error", "Wrong path!");
+            if (!$this->checkExecutableFile()) {
+                return $this->returnMessage("error","Failed to change permissions of shell program");
+            }
+            if (!$this->checkShellProgramExists()) {
+                return $this->returnMessage("error", "Please install shell program!");
+            }
+            
+            $program = $this->getShellProgram();
+            if($remote == 0){
```

Comment:
Does not work for me. After I changed it to `if($remote == "0"){` it worked properly. Could you please test this.